pod-operation/src/state_machine.rs

```diff
@@ -162,13 +167,18 @@ impl StateMachine {
 	/// Perform operations when the pod is loading
 	fn _load_periodic(&mut self) -> State {
 		info!("Rolling Load state");
+
 		State::Load
 	}
 
 	/// Perform operations when the pod is running
 	fn _running_periodic(&mut self) -> State {
 		info!("Rolling Running state");
-		// TODO: add actual decision logic
+		if self.upstream_pressure_transducer.read_pressure() < 126.0
+			|| self.downstream_pressure_transducer.read_pressure() < 126.0
```

Comment:
Chore: As with the other integrations, could we extract these thresholds to global constants (and include an inline comment there with the units of PSI)?
We'll also need to reconfirm with Braking what the upstream threshold should be.

Reply:
Was discussing with braking and decided that monitoring upstream threshold is a bit redundant since the downstream will be slightly more choked and would trigger the fault before an upstream monitor would (if using the same threshold). We can still display the value as part of #44, but for now, let's remove the upstream check.

Reply:
eta 9pm

Reply:
Could we also get an inline comment for the units of PSI on the global value as mentioned, thanks

Reply:
yeah i thin its been in there